src/v2/handler/dataagreement/config_delete_dataagreement.go

```diff
@@ -0,0 +1,62 @@
+package dataagreement
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/bb-consent/api/src/common"
+	"github.com/bb-consent/api/src/config"
+	"github.com/bb-consent/api/src/dataagreement"
+	"github.com/bb-consent/api/src/revision"
+	"github.com/gorilla/mux"
+)
+
+func deleteDataAgreementIdFromDataAttribute() {
```

Comment:
@albinpa Why is this not implemented ?

Reply:
@georgepadayatti Its not implemented because data attributes functionalities are not  yet implemented